src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Calculator/CalculateHelper.cs

```diff
@@ -60,6 +61,13 @@ public static string FixHumanMultiplicationExpressions(string input)
             return output;
         }
 
+        private static string CheckScientificNotation(string input)
+        {
+            var p = @"(-?(\d+(\.\d*)?)|-?(\.\d+))[eE](-?\d+\.*\d*)";
+            var r = "($1 * 10^($5))";
```

Comment:
This line will need to be updated accordingly from the change in capturing groups.
```suggestion
            var r = "($1 * 10^($3))";
```